### PR TITLE
Fixed fish completions in the deb installer

### PR DESCRIPTION
**Pull request by Lucretiel**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -76,7 +76,7 @@ assets = [
     ["target/release/alacritty", "usr/local/bin/", "755"],
     ["alacritty.desktop", "usr/share/applications/", "644"],
     ["alacritty-completions.bash", "usr/share/bash-completion/completions/alacritty", "644"],
-    ["alacritty-completions.fish", "usr/share/fish/completions/alacritty", "644"],
+    ["alacritty-completions.fish", "usr/share/fish/completions/alacritty.fish", "644"],
     ["alacritty-completions.zsh", "usr/share/zsh/functions/Completion/alacritty", "644"],
```
Comment (chrisduerr) [inline Cargo.toml]: Would you mind changing this too? I believe the correct syntax for zsh is `_alacritty` instead of `alacritty`.

Comment (Lucretiel) [reply Cargo.toml]: Sure, although would it be more accurate to put it in `.../ Completion/Unix/_alacritty`? Or `.../zsh/vendor-completions/_alacrity`? I don't really know zsh; that's just my guess from looking in those directories

Comment (chrisduerr) [reply Cargo.toml]: Actually don't worry about it. There's already https://github.com/jwilm/alacritty/pull/1503 to address this. I knew this issue sounded familiar. Looks like this is good the way it is, thanks!

**Comment by Lucretiel** [comment]
Hmm, it seems this didn't actually work. Will investigate further.

**Comment by Lucretiel** [comment]
Oh, never mind, it did work. I idiotically forgot that OSX doesn't install .deb files.

**Pull request merged**